From 3b76a9f57ea65a7ebd22395eb5d3fda9cc929e8c Mon Sep 17 00:00:00 2001
From: NP-Hardass <np.hardass@gmail.com>
Date: Tue, 4 Aug 2020 19:35:28 -0400
Subject: [PATCH] eom-metadata-reader: add gdk/gdk.h include to resolve #258

be4fa2808c4e1a36c2c9b58c1267a54cdda6e495, enable wayland support

added "defined(GDK_WINDOWING_x11)" macros without the accompanying gdk
headers
---
 src/eom-metadata-reader.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/eom-metadata-reader.c b/src/eom-metadata-reader.c
index cd796f0..2c93a14 100644
--- a/src/eom-metadata-reader.c
+++ b/src/eom-metadata-reader.c
@@ -23,6 +23,8 @@
 #include <config.h>
 #endif
 
+#include <gdk/gdk.h>
+
 #include "eom-metadata-reader.h"
 #include "eom-metadata-reader-jpg.h"
 #include "eom-metadata-reader-png.h"
-- 
2.26.2

